Description: Change logging on ia_addr message
 Message about error with subcontainers is just debug, not a filtered by
 keyword debug. This (along with logging options set to high in the init 
 script) made snmpd very chatty.
Author: Pik Master <pikmaster@wp.pl>
Origin: upstream, http://sourceforge.net/p/net-snmp/patches/1322/
Bug-Debian: https://bugs.debian.org/684721
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2019-10-10
--- a/snmplib/container.c
+++ b/snmplib/container.c
@@ -332,8 +332,8 @@
     if(x) {
         int rc = x->insert(x,k);
         if(rc)
-            snmp_log(LOG_DEBUG,"error on subcontainer '%s' insert (%d)\n",
-                     x->container_name ? x->container_name : "", rc);
+            DEBUGMSGTL(("container:insert_helper", "error on subcontainer '%s' insert (%d)\n",
+                     x->container_name ? x->container_name : "", rc));
         else {
             rc = CONTAINER_INSERT_HELPER(x->next, k);
             if(rc)
